File: _includes/SampleCode/SampleCode/Samples/Auth/Initialization.cpp
#include "pch.h"

using namespace Windows::Foundation::Collections;
using namespace Platform;
using namespace concurrency;
using namespace winsdkfb;
using namespace winsdkfb::Graph;

namespace SampleCode
{
    void Initialization()
    {
        FBSession^ sess = FBSession::ActiveSession;
        sess->FBAppId = "<Facebook App ID>";
        sess->WinAppId = "<Windows or Windows Phone Store ID depending on the target device";
    }
}

File: _includes/SampleCode/SampleCodeCs/Samples/Auth/Initialization.cs
using winsdkfb;

namespace SampleCode
{
    public static partial class SampleFunc
    {
        static void Initialization()
        {
            FBSession sess = FBSession.ActiveSession;
            sess.FBAppId = "<Facebook App ID>";
            sess.WinAppId = "Windows or Windows Phone Store ID depending on target device";
        }
    }
}
